x/ccv/consumer/module.go

```diff
@@ -113,6 +113,12 @@ func (AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 // RegisterServices registers module services.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	consumertypes.RegisterQueryServer(cfg.QueryServer(), am.keeper)
+
+	migrator := keeper.NewMigrator(am.keeper, am.paramSpace)
+	err := cfg.RegisterMigration(am.Name(), 1, migrator.MigrateParams)
```

Comment:
don't we need to register the existing consumer migrations here as well?

Reply:
After rebasing on `main` to pull in throttling changes we need to change `1 -> 2`.
That means that the `ConsensusVersion` on the consumer should become 3 instead of 2 (which it is now because we didn't get the throttling changes in yet).